packages/app-elements/src/ui/composite/TopRightToolbar.tsx

```diff
@@ -0,0 +1,78 @@
+import { withSkeletonTemplate } from '#ui/atoms/SkeletonTemplate'
+import { type DropdownItemProps } from '#ui/composite/Dropdown/DropdownItem'
+import {
+  Toolbar,
+  type ToolbarItem,
+  type ToolbarProps
+} from '#ui/composite/Toolbar'
+
+type ToolbarButton = Omit<ToolbarItem, 'dropdownItems'>
+
+export interface TopRightToolbarProps {
+  /**
+   * Toolbar first level `Button`s.
+   */
+  buttons?: ToolbarButton[]
+  /**
+   * Toolbar dropdown items sets. They are rendered in a generated `Dropdown`.
+   */
+  dropdownItems?: DropdownItemProps[][]
+}
+
+/**
+ * This component renders a simple top right bar with just `buttons` and `dropdownItems` optional props.
+ * It will use the `Toolbar` component UI to render properly the given items providing automated responsive behaviors to reorganize them.
+ */
+export const TopRightToolbar = withSkeletonTemplate<TopRightToolbarProps>(
+  ({ buttons, dropdownItems }) => {
+    const toolbarItems: ToolbarProps['items'] = []
+    const dropdownItemsToAdd: DropdownItemProps[] = []
+
+    buttons?.forEach((button) => {
+      // Desktop only button is pushed to the first level of the toolbar
+      const buttonWithCss = {
+        ...button
+      }
+      buttonWithCss.className = 'hidden md:flex'
+      toolbarItems.push(buttonWithCss)
```

Comment:
what do you think about the following?
```suggestion
      toolbarItems.push({
        ...button,
        className: 'hidden md:flex'
      })
```

same could be done for `dropdownItemWithCss`